### PR TITLE
Fixes Sign out from Simulator

### LINKED ISSUE #1124: Sign Out from Simulator Not Working
**Describe the bug**
Whenever the user wants to sign out through the link in simulator it shows `The record you wish access could not be found `
**To Reproduce**
1. Go to 'New UI on Gitpod or beta.circuitverse.org'
2. Click on 'simulator'
3. Go to 'sign out'

**Expected behavior**
It must sign out normally and show message Signed out successfully.

**Screenshots**
Issue10

**Desktop (please complete the following information):**
 - OS: [Windows 10]
 - Browser [Chrome]

**Are you working on this issue? (Yes/No)**
Yes

### PR DESCRIPTION
**Pull request by Shivansh2407**
Fixes #1124 

#### Describe the changes you have made in this PR -
The sign out from simulator is now working. 

### Screenshots of the changes (If any) -
patch-32

### CODE REVIEW

**Comment by Shivansh2407** [comment]
@satu0king , @tachyons , @nitin10s Please review 👍

**Comment by Shivansh2407** [comment]
Furthermore some things that need to be fixed along with this are : 
1. The Dropdown must be more towards the left .
Currently it looks like it is going out of the screen. See Screenshot below.
Capture

2. When the User is on Simulator Page and then clicks on sign in then after signing in , the user remains on the main page and is not redirected back to the Simulator . 

I would like to know your views on this @satu0king , @tachyons , @nitin10s .

**Comment by Shivansh2407** [comment]
@satu0king Please review and Merge this PR 👍

File: app/assets/javascripts/simulator.js [hunk 1]
```diff
@@ -1,2 +1,5 @@
+//= require rails-ujs
+//= require_tree .
```
Comment (tachyons) [inline app/assets/javascripts/simulator.js]: Do we really need require tree here ?

File: app/assets/javascripts/simulator.js [hunk 2]
```diff
@@ -1,2 +1,5 @@
...
 //= require jquery
-//= require bootstrap/dist/js/bootstrap.min.js
+//= require jquery_ujs
```
Comment (tachyons) [inline app/assets/javascripts/simulator.js]: We already have rails_ujs, no need for jquery_ujs

**Comment by Shivansh2407** [comment]
@tachyons I have removed `jquery_ujs` and `tree .` . It is working fine . Please review 👍

**Pull request merged**